web/php/utils/WikijumpController.php

```diff
@@ -1,8 +1,15 @@
 <?php
 
-use DB\Site;
-use DB\SitePeer;
-use DB\MemberPeer;
+namespace Wikidot\Utils;
+
+
+use Ozone\Framework\Database\Criteria;
+use Ozone\Framework\Ozone;
+use Ozone\Framework\WebFlowController;
+use Wikidot\DB\OzoneUser;
+use Wikidot\DB\Site;
+use Wikidot\DB\SitePeer;
+use Wikidot\DB\MemberPeer;
 
 abstract class WikijumpController extends WebFlowController
```

Comment:
I'm going to rename this back to `WikidotController` to remove ambiguity that this is legacy code.